src/app/external-log-in/external-log-in/external-log-in.component.html

```diff
@@ -0,0 +1,49 @@
+<div class="row">
+  <h4>{{ 'external-login.confirmation.header' | translate}}</h4>
+</div>
+<div class="row justify-content-center">
+  <ng-container *ngComponentOutlet="getExternalLoginConfirmationType(); injector: objectInjector;">
+  </ng-container>
+</div>
+<ds-alert class="row mt-2" [type]="AlertTypeEnum.Info" [attr.data-test]="'info-text'">
+  {{ informationText }}
+</ds-alert>
+<div class="row justify-content-center">
+  <div class="col-4 d-flex justify-content-end align-items-center">
+    <ng-container *ngIf="registrationData.email; else provideEmail">
+      <ds-confirm-email [registrationData]="registrationData" [token]="token"></ds-confirm-email>
+    </ng-container>
+    <ng-template #provideEmail>
+      <ds-provide-email [registrationId]="registrationData.id" [token]="token"></ds-provide-email>
+    </ng-template>
+  </div>
+  <div class="col-1 align-items-center d-flex justify-content-center">
+    <h4 class="mt-2">or</h4>
```

Comment:
This looks to be untranslated.   It probably should have an i18n key.  (Though I admit I'm not understanding why "or" is a header because I haven't been able to test this yet. It seems odd to me that this would be a header, as it doesn't seem like useful information)